release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -96,7 +96,14 @@ Previously, {op-system} DHCP kernel parameters were not working as expected beca
 ====
 Only enable multipathing with kernel arguments within a machine config as documented. Do not enable multipathing during installation.
 
-For more information, see "Enabling multipath with kernel arguments" in xref:../post_installation_configuration/machine-configuration-tasks.adoc#rhcos-enabling-multipath_post-install-machine-configuration-tasks[Post-installation machine configuration tasks].
+For more information, see "Enabling multipath with kernel arguments on {op-system}" in xref:../post_installation_configuration/machine-configuration-tasks.adoc#rhcos-enabling-multipath_post-install-machine-configuration-tasks[Post-installation machine configuration tasks].
```

Comment:
I only see this format in the ISG for references to other collections, so I don't think it's right to use here. I think that you're setting enough context in the surrounding text to just use the anchor title for the link text, like:
```suggestion
For more information, see xref:../post_installation_configuration/machine-configuration-tasks.adoc#rhcos-enabling-multipath_post-install-machine-configuration-tasks[Enabling multipath with kernel arguments on {op-system}].
```

If you agree, please update the other link, too.